src/WinGet.RestSource/Models/Arrays/UnsupportedOSArchitectures.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="UnsupportedOSArchitectures.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.WinGet.RestSource.Models.Arrays
+{
+    using System;
+    using Microsoft.WinGet.RestSource.Models.Core;
+    using Microsoft.WinGet.RestSource.Validators.EnumValidators;
+
+    /// <summary>
+    /// UnsupportedOSArchitectures.
+    /// </summary>
+    public class UnsupportedOSArchitectures : ApiArray<string>
+    {
+        private const bool Nullable = true;
+        private const bool Unique = true;

```

Comment:
Why does this have no max? I see its no max in the schema as well. seems like a bug?

Reply:
Ah. it is listed as an enum, fair enough